phf/examples/uncased-example/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "unicase-example"
```

This causes warnings when building due to a name conflict.

```
warning: skipping duplicate package `unicase-example` found at `~/.cargo/git/checkouts/rust-phf-1a7370888ae77131/3a86c8a/phf/examples/unicase-example`
```

```suggestion
name = "uncased-example"
```